addons/spreadsheet_dashboard/static/tests/dashboard/dashboard_action.test.js

```diff
@@ -40,6 +40,27 @@ function getServerData(spreadsheetData) {
     return serverData;
 }
 
+async function openCogMenu() {
+    const cogIcon = getFixture().querySelector(
+        ".o_search_panel_category_value .o_cog_menu_container .fa-cog"
+    );
+    await click(cogIcon);
+    await animationFrame();
```

Comment:
IIRC calling `await` on click is now useless as click is not synchronous.

The "correct" way to do a click + await an animation frams is now `await contains(selector).click()`

Reply:
I just tried it, but it's not working in my case. The likely reason is that the element's visibility is hidden. The `click` method (which is synchronous and doesn't require awaiting) used here, implicitly hovers over the element before clicking on it.